cwl/src/main/scala/wdl4s/cwl/package.scala

```diff
@@ -53,4 +69,6 @@ package object cwl extends TypeAliases {
 
   type Yaml = String
```

Comment:
To be clear, a case class wrapping a string that has a validating constructor would be superior, but that happens as part of yaml parsing so I'm not sure about doing it now.